test/test_ops.py

```diff
@@ -662,9 +663,25 @@ def test_slice_errors(self):
     a = Tensor.ones(4, 3)
     with self.assertRaises(IndexError):
       a[1, 77, 77, 77]  # IndexError: (finds too many indices before the out of bounds)
-      a[1, 77]  # IndexError: (out of bounds).
       a[0, -77]
       a[..., ...] # IndexError: only single ellipsis
+    with self.assertRaises(IndexError): # no floats
+      a[1.1]
```

Comment:
for multiple test cases, i think you would need to separate each into its own `with` block. otherwise it only tests the first one